Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -85,10 +85,11 @@ extension VisualInstruction {
     /// Returns whether the `VisualInstruction`’s maneuver image should be flipped according to the driving side.
     public func shouldFlipImage(side: DrivingSide) -> Bool {
         switch maneuverType ?? .turn {
+        case _ where maneuverDirection == .uTurn:
+            return side == .right
         case .takeRoundabout,
              .turnAtRoundabout,
-             .takeRotary,
-             _ where maneuverDirection == .uTurn:
```

Comment:
Removing this bit causes non-U-turn roundabout and rotary maneuvers to go down this code path, making the roundabout icon’s flipping dependent on the driving side too. That makes some sense, but we should double-check the possible scenarios just in case. If it does check out, we can simplify this switch statement into a simple if/else statement.